content/200-concepts/100-components/03-prisma-migrate/050-prisma-migrate-flows.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Migration flows'
+metaTitle: 'Migration flows (Concepts)'
+metaDescription: 'Prisma Migrate is a declarative data modeling and schema migration tool that is available via the Prisma CLI.'
+---
+
+<TopBlock>
+
+</TopBlock>
+
+## Modifying a migration
+
+You can choose to change the behavior of a migration **when you create it**. For example, Prisma Migrate cannot automatically detect that you want to _rename_ a field - instead, it will add a new field and remove the old one. To rename the field instead (for example, `name` &rarr; `fullname`), you must create and modify a **draft migration**:
+
```

Comment:
Maybe we can mention here that often you will get warnings when this might be necessary. About dataloss or unexecutable migrations.